src/server/node.rs

```diff
@@ -131,6 +131,18 @@ where
         }
     }
 
+    pub fn bootstrap_store_id(&mut self, engines: Engines<RocksEngine, ER>) -> Result<()> {
+        let mut store_id = self.check_store(&engines).unwrap();
+        if store_id == INVALID_ID {
+            store_id = self.bootstrap_store(&engines).unwrap();
+            fail_point!("node_after_bootstrap_store", |_| Err(box_err!(
+                "injected error: node_after_bootstrap_store"
+            )));
+        }
+        self.store.set_id(store_id);
+        Ok(())
+    }
```

Comment:
How about move this logic into `fn bootstrap_store`?

Reply:
I replace with a more proper function name.

Reply:
I think it is better to merge these two functions into one since there are no separated use cases of them and they serve the same purpose: initialize the store id

Reply:
updated.